src/flexalloc_mm.c

```diff
@@ -493,6 +494,26 @@ fla_init(struct fla_geo *geo, struct xnvme_dev *dev, struct xnvme_dev *md_dev, v
   return 0;
 }
 
+static int
+fla_mkfs_make_p_consistent(struct xnvme_dev *dev, struct fla_mkfs_p ** p)
```

Comment:
Do we really need the code in this function? ZNS works fine if you send the right parameters.

Reply:
This patch is just to activate testing. So putting this logic in testing common is probably the correct place for it at the point